src/components/base/outline-alert/outline-alert.ts

```diff
@@ -0,0 +1,64 @@
+import { html, TemplateResult, CSSResultGroup } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import componentStyles from './outline-alert.css.lit';
+import { OutlineElement } from '../outline-element/outline-element';
+
+export type AlertSize = 'small' | 'large';
+export const alertSizes: AlertSize[] = ['small', 'large'];
+
+export type AlertStatusType = 'information' | 'warning' | 'error' | 'success';
+export const alertStatusTypes: AlertStatusType[] = [
+  'information',
+  'warning',
+  'error',
+  'success',
+];
+
+/**
+ * The Outline Alert component
+ *
+ * @element outline-alert
+ * @slot default - The alert contents
+ * @slot outline-alert--header - The header in the alert
+ */
+@customElement('outline-alert')
+export class OutlineAlert extends OutlineElement {
+  static styles: CSSResultGroup = [componentStyles];
+
+  render(): TemplateResult {
+    // The `body` wrapper is used to avoid styles (like border) that are preventing us from styling `:host`.
+    return html`
+      <div id="body" role="${this.isInteractive ? 'alertdialog' : 'alert'}">
+        ${this.shouldShowIcon === true
+          ? html`
+              <div id="icon">
+                <!--@todo include icon when we have that ready.-->
+              </div>
+            `
+          : html``}
+        ${this.size === 'large'
+          ? html`
+              <div id="header">
```

Comment:
Should this use the `<outline-header>`

Reply:
Thanks for the question. I'm not sure.

In the storybook examples I left it as a slot, so we could pass in an `<outline-header>` if desired.

The trade-off there seems to be a little more work on the consumer of the component for the flexibility to define the header level / visual level.

I suppose we could always look to see if the slotted component was an `<outline-header>`, text, or something else and then only wrap text in the `<outline-header>`. What levels would we use then as defaults?

Anyone have ideas?

Reply:
Good point.  I saw the `this.statusType` and was thinking that was the text that was passed, but realize that is just a placeholder and we should be using content in the slot so that might make more sense.  One reason I guess would be to keep all headings the same style.

Reply:
I agree with Chris on this one, I think this should be an `outline-heading` with the statusType rendered inside of it. I don't think there is much benefit to allowing a consumer to place there own element here. If someone wanted to do that they could extend this element and overwrite the `largeTemplate()` method I'd mentioned above.

Reply:
After a while on a project with the enhanced editor I think the slot is a better idea.